front/app/modules/commercial/content_builder/admin/components/CraftComponents/Image/index.tsx

```diff
@@ -58,10 +62,12 @@ const ImageSettings = injectIntl(({ intl: { formatMessage } }) => {
   useEffect(() => {
     if (imageUrl) {
       (async () => {
+        eventEmitter.emit(IMAGE_UPLOADING_EVENT, true);
         const imageFile = await convertUrlToUploadFile(imageUrl);
         if (imageFile) {
           setImageFiles([imageFile]);
         }
+        eventEmitter.emit(IMAGE_UPLOADING_EVENT, false);
       })();
```

Comment:
Not sure if this is the best way to handle this? I know we need to use eventEmitters (like the one we use for errors since CraftJS doesn't play well with passing setState functions), however I'm not quite sure where best to put them.